src/applications/static-pages/sass/static-pages.scss

```diff
@@ -84,3 +84,48 @@ $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
     }
   }
 }
+
+//=============================
+// Up To Top Button Styles 
+//=============================
+.vsa-top-button-container {
+  bottom: 0;
+  left: 0;
+  pointer-events: none;
+  position: fixed;
+  width: 100%;
+  &.vsa-top-button-container-relative {
+    position: relative;
+  }
+
+  .usa-width-three-fourths.vsa-top-button-container-relative { 
+    width: 100%;
+  }
+
+  .usa-content {
+    position: relative;
+  }
+
+  .vsa-top-button {
+    bottom: 0;
+    margin: 0;
+    opacity: 0;
+    pointer-events: none;
+    position: absolute;
+    transition: all 250ms ease-in;
+    right: 0;
+    width: auto;
+    &.vsa-top-button-transition-in {
+      pointer-events: all;
+      opacity: 1;
+      transform: translateY(-12px);
+    }
+    &.vsa-top-button-transition-out {
+      opacity: 0;
+      transform: translateY(2px);
+    }
+    &.vsa-top-button-transition-reset {
+      transform: translateY(0px);
+    }
```

Comment:
This transition looks cool, but does the design spec call out this out as a requirement? Asking because it looks like it adds a fair amount of complexity into the JS